src/mavsdk/core/mavsdk_impl.cpp

```diff
@@ -119,6 +119,49 @@ std::vector<std::shared_ptr<System>> MavsdkImpl::systems() const
     return systems_result;
 }
 
+std::optional<std::shared_ptr<System>> MavsdkImpl::first_autopilot(double timeout_s)
+{
+    {
+        std::lock_guard<std::recursive_mutex> lock(_systems_mutex);
+        if (!_systems.empty()) {
+            return {_systems[0].second};
```

Comment:
Shouldn't it go through all the systems and return the first autopilot? This will just return the first system, right?

Reply:
Whoops.